corehq/apps/hqadmin/management/commands/clone_domain.py

```diff
@@ -317,15 +325,19 @@ def save_couch_copy(self, doc, new_domain=None):
         self.log_copy(doc.doc_type, old_id, new_id)
         return old_id, new_id
 
-    def save_sql_copy(self, model, new_domain):
-        old_id = model.id
+    def save_sql_copy(self, model, new_domain, domain_is_pk=False):
+        old_id = model.domain if domain_is_pk else model.id
```

Comment:
you could just change this to `model.pk` which will work for all models (except ones with composite primary keys)